apps/anon-message-client/.env.local.example

```diff
@@ -0,0 +1,10 @@
+# The port on which this server runs. We recommend `4000` for convenient
+# local integration with the rest of the services that run within this
+# repository.
+NEXT_PUBLIC_PORT=4000
+
+# The passport server needs to know where it itself can be accessed in order
+# to be able to create URLs that point to itself.
+NEXT_PUBLIC_PASSPORT_SERVER_URL="https://dev.local:3002"
+NEXT_PUBLIC_PASSPORT_CLIENT_URL="https://dev.local:3000"
+NEXT_PUBLIC_IS_LOCAL_HTTPS=true
```

Comment:
we'll need to think of a better way to do HTTPS dev - I think CLI environment variables would be better than .env file

anyway, no action item right now - just something to think about